Musicca/app/src/main/java/com/example/musicca/adapters/CurrentPlaylistAdapter.java

```diff
@@ -45,6 +47,7 @@
 
     private static final String TAG = "Queue";
 
+    public List<String> sortedSongObjectIds;
```

Comment:
Why does this need to be public? In general it's discouraged to expose the members directly. You can create getter methods for them.